Cabal/Distribution/Simple/Utils/Json.hs

```diff
@@ -0,0 +1,46 @@
+-- | Utility json lib for Cabal
+-- TODO: Remove it again.
```

Comment:
please explain. Make an issue, link to it.

TODOs in the code are almost never fixed "immediately".